action.yml

```
@@ -0,0 +1,6 @@
+name: container-diff
+
+runs:
+  using: 'docker'
+  image: 'action/Dockerfile'
```

We may also need something like this for passing arguments into the action/container:

```suggestion
inputs:
  argument:
    required: true
    description: "String of arguments to pass to the container-diff command"
    default: help

runs:
  using: 'docker'
  image: 'action/Dockerfile'
  args:
    - ${{ inputs.argument }}
```